commands/ps/scale.js

```diff
@@ -22,19 +22,41 @@ function * run (context, heroku) {
     }))
   }
 
-  let changes = parse(context.args)
-  if (changes.length === 0) {
-    let formation = yield heroku.get(`/apps/${app}/formation`)
+  const displayFormation = co.wrap(function * () {
+    const formation = yield heroku.get(`/apps/${app}/formation`)
     if (formation.length === 0) throw emptyFormationErr(app)
     cli.log(formation.map((d) => `${d.type}=${d.quantity}:${d.size}`).sort().join(' '))
-  } else {
-    yield cli.action('Scaling dynos', {success: false}, co(function * () {
-      let formation = yield heroku.request({method: 'PATCH', path: `/apps/${app}/formation`, body: {updates: changes}})
+  })
+
+  const autoscalingCheck = co.wrap(function * () {
+    let monitors
+    try {
+      const appID = (yield heroku.get(`/apps/${app}`)).id // TODO: metrics api will eventually not require app ID
+      monitors = yield heroku.get(`/apps/${appID}/formation/web/monitors`, {host: 'api.metrics.heroku.com'})
+    } catch (err) { // skip autoscaling check if any http errors
```

Comment:
if it fails because of a 5xx, then we will be notified that it happens and have records of it. On your end, you may want to do an info/warning-level report to Rollbar?